package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -380,6 +394,16 @@ dhcp_add() {
 	config_get options "$cfg" options
 	config_get_bool dynamicdhcp "$cfg" dynamicdhcp 1
 
+	if [ $ADD_LOCAL_HOSTNAME -gt 0 ] ; then
+		local dashedname="$( echo $ifname | sed 's/\./-/g' )"
```

Comment:
You can use `${ifname//./-}` here to avoid the need for spawning a `sed` subshell.